misc/taskmetadata-validator/taskmetadata-validator.go

```diff
@@ -243,6 +251,11 @@ func main() {
 		}
 	}
 
+	if containerMetadata.Volumes == nil {
```

Comment:
Can you check some of the fields that were specified in the task definition?

Reply:
the docker name and source are difficult to be verified, since the volume name is created with some task information and random ID at the end. Hence checking for `nil` here. Do you have other suggestions?